### ISSUE TITLE
i18n missing

### ISSUE THREAD

**Issue by nwittstruck** [magiclabs/spree_payone#3]
I'm seeing the following exception in the checkout workflow after installing the current master:
`
<Span Class="Translation Missing" Title="Translation Missing: En.Spree.Order State.Payment Redirect">Payment Redirect</Span>
`
This locale is neither defined for english, nor german. I can create a PR, I am just unsure if this hasn't been noticed before? Is something else wrong with my installation? Depending on the meaning of the step, should usually it be hidden?

**Comment by tvdeyen** [comment]
Yes, this should be hidden from the checkout views. It would be great if you could provide a pull request. Thanks

**Comment by mamhoff** [comment]
The payment redirect step is now implemented as a step in the order state machine that models the checkout flow. That's why this missing translation appears. Ideally, the logic executed there should be implemented as a `before` or `after` hook on the `complete` step. If you want to look into that, that'd be great!

**Comment by nwittstruck** [comment]
Ah okay, it's a bit more difficult than just adding the missing i18n. I don't have access to the Payone API yet, but I can take a look as soon as I do.
